# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../gatling/lowfat-0.25.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./gatling.c.orig	2007-04-12 07:31:12.000000000 +0000
+++ ./gatling.c	2007-04-12 07:31:38.000000000 +0000
@@ -865,24 +865,6 @@
   return (c!='"' && c!='%' && c>=' ' && c!='+');
 }
 
-unsigned long fmt_urlencoded(char* dest,const char* src,unsigned long len) {
-  register const unsigned char* s=(const unsigned char*) src;
-  unsigned long written=0,i;
-  for (i=0; i<len; ++i) {
-    if (!issafe(s[i])) {
-      if (dest) {
-	dest[written]='%';
-	dest[written+1]=fmt_tohex(s[i]>>4);
-	dest[written+2]=fmt_tohex(s[i]&15);
-      }
-      written+=3;
-    } else {
-      if (dest) dest[written]=s[i]; ++written;
-    }
-  }
-  return written;
-}
-
 void catencoded(array* a,char* s) {
   unsigned int len=str_len(s);
   char* buf=alloca(fmt_urlencoded(0,s,len));
